### DISCUSSION TITLE
about upgrade BIOS,i do not know why

### DISCUSSION THREAD

**Discussion by zzduci** [sileshn/Ryzentosh#44] [Q&A] [answered]
I upgraded to the latest macOS 15.1 using your method, but after upgrading the motherboard BIOS to the latest version, my system won't boot. Downgrading the BIOS to the original version allows it to boot again. Is there a better way to get it to boot normally besides downgrading?

**Comment by sileshn** [comment]
So obviously something in the bios is making it not boot. I cannot help with that.

**Comment by zzduci** [comment]
The option settings of the new BIOS are exactly the same as those of the old BIOS.

**Comment by zzduci** [comment, marked as answer]
i know why it does not  working,   i fix it by myself,   maybe can help someone  
![f5204fb88f104fd28a488bac4746d83](https://github.com/user-attachments/assets/18c05d15-5d07-4c76-9e0f-f96ffe66df4d)
![image](https://github.com/user-attachments/assets/74809bd0-26c2-49b2-8ded-fe8b27bea139)

![image](https://github.com/user-attachments/assets/8cf314c7-4e09-4056-a540-9efb089d40fc)
it is working now
